mne/tests/test_epochs.py

```diff
@@ -4286,7 +4286,7 @@ def test_pick_types_reject_flat_keys():
 
     assert sorted(epochs.reject.keys()) == ["eeg", "eog", "grad", "mag"]
     assert sorted(epochs.flat.keys()) == ["eeg", "eog", "grad", "mag"]
-    epochs.pick_types(meg=True, eeg=False, ecg=False, eog=False)
+    epochs.pick(picks="meg")
```

Comment:
OK, it looks like this is a real issue. It seems like `pick()` does *not* remove keys from reject/flat anymore (in contrast to `pick_types()`). What do you suggest?

Reply:
That seems like a bug